tests/helpers/perf_run_helper.rb

```diff
@@ -93,13 +94,42 @@ def perf_teardown
   # @example:
   #   create_perf_archive_root
   #
-  def create_perf_archive_root
+  def create_perf_archive_area
```

Comment:
:-1: for this breaking change.
The intent of this method has not changed.  The `@archive_root` directory is still created by this method it just does more work.  I do not think there is enough change to warrant altering this method name.

Reply:
Since the method is only used internal to this file, it didn't feel like a breaking change.  And while the old name is still true, if you read the code for the first time with no background history, you would think the name is wrong.  That is what I was trying to fix... we can discuss in the parking lot